pom.xml

```diff
@@ -511,11 +511,47 @@ under the License.
                 <pomExclude>pom.xml</pomExclude>
               </pomExcludes>
               <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
+              <filterProperties>
+                <repository.proxy.url>${repository.proxy.url}</repository.proxy.url>
+              </filterProperties>
+              <addTestClassPath>true</addTestClassPath>
+
               <goals>
                 <goal>site</goal>
               </goals>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>mrm-maven-plugin</artifactId>
+            <version>1.6.0</version>
```

Comment:
This is set in parent

Reply:
I couldn't find the mrm-maven-plugin defined in a parent pom.

Reply:
Weird, I bet I have seen it. Must be something else where I bumped this plugin. Never mind